packages/calcite-components/src/components/time-picker/assets/time-picker/t9n/messages.json

```diff
@@ -5,6 +5,9 @@
   "meridiem": "AM/PM",
   "meridiemDown": "Decrease AM/PM",
   "meridiemUp": "Increase AM/PM",
+  "fractionalSecond": "Fractional second",
```

Comment:
Can you also update `messages_en.json`? Both need to be in sync. cc @anveshmekala